dev/MRTCore/mrt/Microsoft.ApplicationModel.Resources/projection/Microsoft.ApplicationModel.Resources.Projection.csproj

```diff
@@ -8,12 +8,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Windows.CsWinRT" Version="1.0.1" />
+    <PackageReference Include="Microsoft.Windows.CsWinRT" Version="1.1.0" />
   </ItemGroup>
 
   <ItemGroup>
-    <FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" RuntimeFrameworkVersion="10.0.18362.10" />
-    <FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" TargetingPackVersion="10.0.18362.10" />
+    <FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" RuntimeFrameworkVersion="10.0.18362.12" />
+    <FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" TargetingPackVersion="10.0.18362.12" />
```

Comment:
Since you're pinning the version of .NET 5 to be used via global.json, do you know if these are still necessary? 

@Scottj1s does the global.json make this superfluous in terms of ensuring that the .NET 5.0.1 dependency is satisfied?